frontend/templates/product/server.tsx

```diff
@@ -39,13 +38,6 @@ export const getServerSideProps: GetServerSideProps<ProductTemplateProps> =
       );
 
       if (product?.__typename === 'ProductRedirect') {
-         const destination = new URL(product.target, ifixitOrigin);
-         const requestParams = new URL(urlFromContext(context)).searchParams;
-         requestParams.forEach((value, key) => {
-            if (!destination.searchParams.has(key)) {
-               destination.searchParams.append(key, value);
-            }
-         });
```

Comment:
I've removed this because the result of the computation (`destination`) is not being used as a redirect destination.
Do we want to fix the logic or should we continue without it, aka keep the current behaviour?

cc @masonmcelvain

Reply:
We want to add request query parameters to the redirect, so let's fix the logic. Good catch!